mycollab-esb/src/main/java/com/mycollab/module/user/esb/ResetUserPasswordCommand.kt

```diff
@@ -60,16 +61,14 @@ class ResetUserPasswordCommand(private val extMailService: ExtMailService,
             contentGenerator.putVariable("urlRecoveryPassword", recoveryPasswordURL)
             contentGenerator.putVariable("copyRight", LocalizationHelper.getMessage(locale, MailI18nEnum.Copyright,
                     DateTimeUtils.getCurrentYear()))
-            val recipient = MailRecipientField(user.email, user.username)
-            val recipientFields = listOf(recipient)
-
+           
             extMailService.sendHTMLMail(applicationConfiguration.notifyEmail, applicationConfiguration.siteName,
-                    recipientFields,
+                    Collections.singletonList(MailRecipientField(user.username, user.username)),
```

Comment:
We should use listOf instead of Collections.singletonList. of course, both works but I prefer to use the existing Kotlin API in Kotlin source instead of Java library. We can write the code like listOf(MailRecipientField(user.username, user.username))

Reply:
ok